forcats.Rmd

```diff
@@ -296,17 +296,17 @@ countries_in_1896 %>%
   levels()  
 ```
 ### Reordering levels by other variables
-The functions presented in this section bare great similarity to the `fct_relevel()` function introduced in the beginning of this section.
-`fct_relevel()` allows for a direct manipulation of the levels by passing the new order to the `levels` keyword argument. In this regard `fct_reorder()` and `fct_reorder2()` are different. Here the levels are reordered according to the result of a function applied to one vector `x` in case of `fct_reorder()` and two vectors `x` and `y` in case of `fct_reorder2()`.
+The functions presented in this section bare great similarity to the `fct_relevel()` function introduced in the beginning of this chapter.
+`fct_relevel()` allows a direct manipulation of the levels by passing the new order to the `levels` keyword argument. At this point `fct_reorder()` and `fct_reorder2()` are different. Here, the levels are reordered according to the result of a function applied to one vector `x` in case of `fct_reorder()` and two vectors `x` and `y` in case of `fct_reorder2()`.
 
-**Note:** We want to point out a few things that have to be kept in mind when using these functions
+**Note:** We want to point out a few things that have to be kept in mind when using these functions.
 
-: 1. The length of the factor `f` to be reordered has the be equal to the lenght of the vector(s) passed to the function. 
-That is an error will be provoked, if `length(f) != length(x)` or `length(f) != length(y)`.
+: 1. The length of the factor `f` to be reordered has the be equal to the length of the vector(s) passed to the function. 
+There will occur an error, if `length(f) != length(x)` or `length(f) != length(y)`.
```

Comment:
<code>There will <b>be</b> an error ,...</code> ?

Reply:
My proposals:
1) there is an error
2) there will occur an error-message